dll/kernel32.cpp

```diff
@@ -1879,14 +1858,12 @@ namespace kernel32 {
 		if (!value) {
 			return 0;
 		}
-		uint16_t *wideValue = stringToWideString(value);
-		const auto len = wstrlen(wideValue);
+		auto wideValue = stringToWideString(value);
+		const auto len = wstrlen(wideValue.data());
```

Comment:
(could also use wideValue.size() to get the same result as len+1 here)

Reply:
At the end of this function, we're returning "len" (wideValue.size() - 1) - should I be still returning that value? I guess I can just put the same expression there if so

Reply:
It doesn't matter, they are the same. `wideValue` cannot be empty, its size is always `strlen(input) + 1`. `wideValue.size() - 1` is slightly cleaner I think, since it avoids some additional string processing (strlen() walks the string; size() doesn't) and gets rid of a variable.